client/me/billing-history/receipt.jsx

```diff
@@ -74,10 +74,15 @@ class BillingReceipt extends React.Component {
 
 		if ( transaction.pay_part === 'paypal_express' ) {
 			text = translate( 'PayPal' );
-		} else if ( 'NOT STORED' === transaction.cc_type.toUpperCase() ) {
-			text = translate( 'Credit Card' );
+		} else if ( 'XXXX' !== transaction.cc_num.toUpperCase() ) {
```

Comment:
Is the `.toUpperCase()` really necessary here? Later in the file there is another check for `XXXX` without it.

Reply:
nope, indeed. I'll remove it